content/installation/installing-manager.md

```diff
@@ -78,7 +78,7 @@ Services:
 
 ### Configuring the Manager Settings
 
-After you install Cloudify Manager, you can change the settings used by the installation and not reinstall the Cloudify components. The configure command accepts the same CLI settings as the install command, and it reads the config.yaml file for additional settings.
+After you install Cloudify Manager, you can change the settings used by the installation without reinstalling the Cloudify components. The configure command accepts the same CLI inputs as the install command, and it reads the same config.yaml file for additional settings.
```

Comment:
I prefer to keep the previous text.
Do you think it's important to change?

Reply:
I think my version is more correct, but I trust your judgement